reader/src/main/resources/config/base.conf

```diff
@@ -7,6 +7,7 @@ service {
     cert.path = ""
     private.key.path = ""
   }
+  max.message.size = 52428800 # 50MB in bytes
 }
 
 backend {
```

Comment:
same comment applies here. Do we need 'max.message.size' config for the grpc client.

Reply:
This one is for trace reader server

Reply:
I was pointing to the config within backend. It is not shown in this PR. Check the file.